docker-compose/database/db-changes/datasets/0030_add_subjectType_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE subject_study ADD COLUMN subject_type int(11) DEFAULT NULL;
```

Comment:
I have this error, you have to initialize the table in the SQL too :(
qualif-datasets-1  | Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [acquisition_equipment]


Excuse me, I tested locally last time, and not on qualif..